frontend/public/components/pdb/PodDisruptionBudgetField.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { useTranslation } from 'react-i18next';
+import { PodDisruptionBudgetKind, LabelSelector } from '@console/internal/module/k8s';
+import { useK8sWatchResource } from '../utils/k8s-watch-hook';
+import { ResourceLink, DetailsItem } from '../utils';
+import { PodDisruptionBudgetModel } from '../../models';
+
+const AvaiallabilityRequirement: React.FC<AvaiallabilityRequirementProps> = ({ pdb, replicas }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {pdb?.spec?.minAvailable
+        ? t('public~Min Available {{minAvailable}} of {{count}} pod', {
+            minAvailable: pdb.spec.minAvailable ?? 1,
```

Comment:
`?? 1` shouldn't be necessary since we already check that `minAvailable` is set above.